.env.example

```
@@ -514,6 +533,16 @@ GOOGLE_PROJECT_NUMBER=GOOGLE_PROJECT_NUMBER
 # Letta
 LETTA_SERVER_PASSWORD=
 
+# Hermes Agent
+# Dashboard login (username = your install email; password/secret auto-generated).
+# LLM provider keys are NOT set here -
+# run 'docker compose -p localai run --rm hermes setup' or edit ./hermes/.env
+HERMES_USERNAME=
+HERMES_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HERMES_PASSWORD key should go before the HERMES_USERNAME key

(UnorderedKey)